src/snapred/backend/recipe/GenericRecipe.py

```diff
@@ -100,17 +98,9 @@ class FocusSpectraRecipe(GenericRecipe[FocusSpectraAlgorithm]):
     pass
 
 
-class ConvertTableToMatrixWorkspaceRecipe(GenericRecipe[ConvertTableToMatrixWorkspace]):
```

Comment:
removing these generic recipes shouldnt be necessary as the implementation of GenericRecipe uses mantid snapper. 

Please add this to the exclude or move generic recipe under the algorithm package instead.